Riot/Modules/Application/AppDelegate.swift

```diff
@@ -59,6 +59,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let window = UIWindow(frame: UIScreen.main.bounds)
         self.window = window
         
+        // Register pills provider.
+        if #available(iOS 15.0, *) {
+            NSTextAttachment.registerViewProviderClass(PillAttachmentViewProvider.self, forFileType: StringPillsUtils.pillUTType)
```

Comment:
Yes, I'll have to check because it seems that there is a lot of ways to create a RoomViewController, I'm not sure they all use the Coordinator properly 😶

Reply:
True we can move that to `RoomViewController` otherwise